pages/release-process.md

```diff
@@ -146,68 +146,108 @@ When the vote passes on a release candidate, follow the steps below to complete
-   for the release candidate. Also, make sure that the `fluoRC` profile is active while you are testing the release
-   but comment out `<activeProfiles>` section after you are done testing.
-    ```xml
-    <profiles>
-      <profile>
-        <id>fluoRC</id>
-        <repositories>
-          <repository>
-            <id>fluorc</id>
-            <name>fluorc</name>
-            <url>https://repository.apache.org/content/repositories/orgapachefluo-1023/</url>
-          </repository>
-        </repositories>
-        <pluginRepositories>
-          <pluginRepository>
-            <id>fluorcp</id>
-            <name>fluorcp</name>
-            <url>https://repository.apache.org/content/repositories/orgapachefluo-1023/</url>
-          </pluginRepository>
-        </pluginRepositories>
-      </profile>
-    </profiles>
-    <activeProfiles>
-      <activeProfile>fluoRC</activeProfile>
-    </activeProfiles>
-    ```
-1. Clone a project that uses Fluo like the [Fluo Recipes] project below:
+#### Set up your environment
+
+Set up required for all tests:
+
+1. Set the release version, staging repo, and path to release Maven settings:
+   ```shell
+   export RC_VERSION=1.2.0
+   export RC_STAGING=https://repository.apache.org/content/repositories/orgapachefluo-1023/
+   export FLUO_MAVEN=/tmp/fluo-maven.xml
+   ```
+1. Create Maven settings
+   ```shell
+   $ cat <<EOF >$FLUO_MAVEN
+   <settings>
+     <profiles>
+       <profile>
+         <id>fluoRC</id>
+         <repositories>
+           <repository>
+             <id>fluorc</id>
+             <name>fluorc</name>
+             <url>\${env.RC_STAGING}</url>
+           </repository>
+         </repositories>
+         <pluginRepositories>
+           <pluginRepository>
+             <id>fluorcp</id>
+             <name>fluorcp</name>
+             <url>\${env.RC_STAGING}</url>
+           </pluginRepository>
+         </pluginRepositories>
+       </profile>
+     </profiles>
+     <activeProfiles>
+       <activeProfile>fluoRC</activeProfile>
+     </activeProfiles>
+   </settings>
+   EOF
+   ```
+1. If a new release candidate is made, update your staging repo.
+   ```shell
+   $ export RC_STAGING=https://repository.apache.org/content/repositories/orgapachefluo-1024/
+   ```
+
+Set up required to run example Fluo applications
+
+1. Clone [Uno] and fetch Fluo dependencies
+   ```shell
+   git clone https://github.com/astralway/uno.git
+   cd uno
+   ./bin/uno fetch fluo
+   ```
+1. Download release tarball and calculate hash
+   ```shell
+   wget -P downloads/ ${RC_STAGING}/org/apache/fluo/fluo/${RC_VERSION}/fluo-${RC_VERSION}-bin.tar.gz
+   shasum -a 256 downloads/fluo-${RC_VERSION}-bin.tar.gz
+   ```
+1. Set `FLUO_VERSION` and `FLUO_HASH` in `conf/uno.conf`.
+   ```shell
+   vim conf/uno.conf
+   ```
+1. Set up Fluo and your shell
+   ```shell
+   ./bin/uno setup fluo
+   eval "$(./bin/uno env)"
+   ```
+
+### Run the integration tests of projects that use Fluo
```

Comment:
I think this section should go after the phrasecount example with uno. It's really a separate test case, and it is breaking up the uno test case with phrasecount.

Reply:
I reorganized it a little better.  I move this before the the uno / phrasecount examples.